planetscale/database_resource.go

```diff
@@ -206,3 +209,57 @@ func (r *databaseResource) Configure(_ context.Context, req resource.ConfigureRe
 
 	r.client = req.ProviderData.(*planetscale.Client)
 }
+
+func splitDatabaseResourceID(id string) (teamID, _id string, ok bool) {
+	attributes := strings.Split(id, "/")
+	requiredAttributesLength := 2
+	if len(attributes) != requiredAttributesLength {
+		return "", "", false
+	}
+	return attributes[0], attributes[1], true
+}
+
+func (r *databaseResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+	organizationName, databaseName, ok := splitDatabaseResourceID(req.ID)
+	if !ok {
+		resp.Diagnostics.AddError(
+			"Error importing database",
+			fmt.Sprintf("Invalid id '%s' specified. should be in format \"organization_name/database_name\"", req.ID),
+		)
+		return
+	}
+
+	out, err := r.client.Databases.Get(ctx, &planetscale.GetDatabaseRequest{
+		Organization: organizationName,
+		Database:     databaseName,
+	})
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Error reading project",
```

Comment:
Same as above - perhaps log specifically the type of resource instead of `project`?

Reply:
Agree!